lib/hpke/test/CMakeLists.txt

```diff
@@ -12,5 +12,5 @@ target_link_libraries(${TEST_APP_NAME} ${CURRENT_LIB_NAME} bytes doctest::doctes
 
 # Enable CTest
 include(doctest)
-enable_testing()
+add_test(${TEST_APP_NAME} ${TEST_APP_NAME})
```

Comment:
This change seems duplicative with the `doctest_discover_tests()` call below.  Why is it needed?